### PR TITLE
fix(server): empty trash for archived assets

### LINKED ISSUE #12280: Archived deleted images are ignored by empty trash
### The bug

I just found a small bug that images in the trash that were archived before are not deleted by the empty trash button

### The OS that Immich Server is running on

Arch

### Version of Immich Server

v1.113.1

### Version of Immich Mobile App

v1.113.1

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
n/a
```

### Your .env content

```Shell
n/a
```

### Reproduction steps

1. Upload and archive photo
2. Delete archived photo
3. Go to trash, click empty trash and reload page -> image is still there

### Relevant log output

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by alextran1502**
Fix #12280

### CODE REVIEW

File: server/src/services/trash.service.ts [hunk 1]
```diff
@@ -38,9 +38,29 @@ export class TrashService {
   }
 
   async empty(auth: AuthDto): Promise<void> {
-    const assetPagination = usePagination(JOBS_ASSET_PAGINATION_SIZE, (pagination) =>
+    let assetPagination = usePagination(JOBS_ASSET_PAGINATION_SIZE, (pagination) =>
+      this.assetRepository.getByUserId(pagination, auth.user.id, {
+        trashedBefore: DateTime.now().toJSDate(),
+        isArchived: true,
```
Comment (mertalev) [inline server/src/services/trash.service.ts:44]: Use `withArchived` instead without duplicating the loop.

Comment (alextran1502) [reply server/src/services/trash.service.ts]: duh, simple! Thanks

**Review by mertalev — APPROVED** [review]

**Review by jrasm91 — APPROVED** [review]
Can you write an e2e test for this please?

**Comment by alextran1502** [comment]
@jrasm91 Yup and done

**Pull request merged**